src/Morphic-Widgets-Basic/PluggableButtonMorph.class.st

```diff
@@ -918,11 +889,9 @@ PluggableButtonMorph >> lastState: anObject [
 PluggableButtonMorph >> layoutBounds: aRectangle [
 	"Set the bounds for laying out children of the receiver.
 	Update the fillstyle since it may depend on the bounds."
-	
+
 	super layoutBounds: aRectangle.
-	self gradientLook ifTrue: [
-		self fillStyle isOrientedFill
-			ifTrue: [self fillStyle: self fillStyleToUse]]
+	self fillStyle isOrientedFill ifTrue: [ self fillStyle: self fillStyleToUse ]
```

Comment:
remove this line as well since it was within the #ifTrue: branch of gradientLook

Reply:
just remove the whole method